File: ai/swift/detect-gemini.swift
// ruleid: detect-gemini
import GoogleGenerativeAI

// Access your API key from your on-demand resource .plist file (see "Set up your API key" above)
// ruleid: detect-gemini
let model = GenerativeModel(name: "gemini-1.5-flash", apiKey: APIKey.default)

let prompt = "Write a story about a magic backpack."
let response = try await model.generateContent(prompt)
if let text = response.text {
  print(text)
}

File: ai/dart/detect-gemini.dart
// ruleid: detect-gemini
import 'package:google_generative_ai/google_generative_ai.dart';

// Access your API key as an environment variable (see "Set up your API key" above)
final apiKey = Platform.environment['API_KEY'];
if (apiKey == null) {
  print('No \$API_KEY environment variable');
  exit(1);
}

// ruleid: detect-gemini
final model = GenerativeModel(model: 'gemini-1.5-flash', apiKey: apiKey);